node/node.cpp

```diff
@@ -98,7 +98,8 @@ node::do_accept()
                         , self->chaos
                         , std::bind(&node::priv_protobuf_handler, self, std::placeholders::_1, std::placeholders::_2)
                         , self->options->get_ws_idle_timeout()
-                        , [](){});
+                        , [](){}
```

Comment:
Add a todo comment for this lambda?

Reply:
That's the "on cleanup" callback, and no callback is required because node doesn't keep a reference to sessions that wrap incoming connections.